app/src/main/java/org/kiwix/kiwixmobile/downloader/DownloadFragment.java

```diff
@@ -76,6 +82,21 @@ private void updateNoDownloads() {
     }
   }
 
+  public static void showNoWiFiWarning(Context context) {
```

Comment:
Lets have this also be the warning when you go to initialy start a download on wifi. We can fully replace that confirmation box with this setting.